Jenkinsfile

```diff
@@ -532,7 +532,10 @@ pipeline {
                     steps {
                         unitTest timeout_time: 60,
                                  inst_repos: prRepos(),
-                                 inst_rpms: unitPackages()
+                                 inst_rpms: unitPackages(),
+                                 target: 'el8',
+                                 ci_target: 'el8',
+                                 distro_version: '8'
```

Comment:
It would be better to change the stage name to include EL8 and then `parseStageInfo()` should set the variables accordingly, eliminating ballooning of the `Jenkinsfile`.

Reply:
If I do that for this pass, the PRs will not pass required github checks.
Can github checks be setup to look for a regex?

Reply:
> If I do that for this pass, the PRs will not pass required github checks. Can github checks be setup to look for a regex?

There is always a window where GH checks don't match up when changing stage names.  Gatekeepers just need to force-land things for a while after they confirm that it's just a stage naming issue.